test/unit/ajax.js

```diff
@@ -2840,6 +2840,32 @@ if ( typeof window.ArrayBuffer === "undefined" || typeof new XMLHttpRequest().re
 		} );
 	} );
 
+	QUnit.test( "jQuery.append with crossOrigin attribute", function( assert ) {
+		var done = assert.async();
```

Comment:
We typically put all var declarations near the top of the function under one `var` keyword, one declaration per line if immediately assigned, unassigned vars all in one line (the first one). Please see other tests for comparison.

Here I'd expect `assert.expect( 1 );` to be the first line, then an empty line & then all vars declared with one var statement.